File: passwords-manager/src/models/account.ts
import {Application} from "./application";

export class Account {
  idAccount: number;
  name: string;
  password: string;
  idApplication: number;
  application: Application;

  constructor() {
    this.idAccount = 0;
    this.name = '';
    this.password = '';
    this.idApplication = 0;
    this.application = new Application();
  }
}

File: EvalDI2P2P3API/Eval.Domain/Models/Account.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Eval.Domain.Models;

[Table("Account")]
public class Account
{
    [Key]
    public int IdAccount { get; set; }
    
    [Required]
    public string Password { get; set; }
    
    [Required]
    public int IdApplication { get; set; }
    
    [ForeignKey(nameof(IdApplication))]
    public Application? Application { get; set; }
}